src/sentry/api/issue_search.py

```diff
@@ -240,6 +261,21 @@ def convert_search_filter(
                 operator = "IN" if search_filter.operator in EQUALITY_OPERATORS else "NOT IN"
             else:
                 operator = "=" if search_filter.operator in EQUALITY_OPERATORS else "!="
+
+            if search_filter.key.name == "substatus":
+                if not features.has("organizations:issue-states", organization, actor=None):
```

Comment:
```suggestion
                if not features.has("organizations:issue-states", organization):
```

`actor` seems to [default to `None`](https://github.com/getsentry/sentry/blob/25d23493007a2c93740c087e79210915b43c5ef7/src/sentry/features/manager.py#L224) in any case.